### ISSUE TITLE
[BUG] Web interface does not show images / CSS

### ISSUE THREAD

**Issue by JustusRijke** [inventree/InvenTree#2425]
**Describe the bug**
The web interface is incorrect, identical to #1720 

**Steps to Reproduce**
1. Bare metal install (tested on system with Ubuntu 21.04 + PostgreSQL 14)
2. Follow steps in https://inventree.readthedocs.io/en/latest/start/production/
3. Browse to 127.0.0.1:8000

**Expected behavior**
Web interface shown in all its glory at 127.0.0.1:8000

**Deployment Method**
- [ ] Docker
- [X] Bare Metal

**Version Information**
InvenTree Version | 0.6.0 dev  Development Version
  | Commit Hash | 8814868fc
  | Commit Date | 2021-12-05
  | InvenTree Documentation | https://inventree.readthedocs.io/en/latest
  | API Version | 21
  | Python Version | 3.9.5
  | Django Version | 3.2.5

Tried starting the background worker manually (e.g. 'python manage.py qcluster'). Worker starts without problems, but this has no effect on the web interface.

**Logs**
`/cluster.err.log`:
```
15:06:30 [Q] INFO Process-1:1 processing [mars-burger-mountain-helium]
15:06:30 [Q] ERROR Failed [mars-burger-mountain-helium] - Model class django.contrib.sessions.models.Session doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS. : Traceback (most recent call last):
  File "/home/inventree/env/lib/python3.9/site-packages/django_q/cluster.py", line 436, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/home/inventree/src/InvenTree/InvenTree/tasks.py", line 240, in delete_expired_sessions
    from django.contrib.sessions.models import Session
  File "/home/inventree/env/lib/python3.9/site-packages/django/contrib/sessions/models.py", line 10, in <module>
    class Session(AbstractBaseSession):
  File "/home/inventree/env/lib/python3.9/site-packages/django/db/models/base.py", line 113, in __new__
    raise RuntimeError(
RuntimeError: Model class django.contrib.sessions.models.Session doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.

15:08:30 [Q] INFO Process-1:2 processing [mars-burger-mountain-helium]
15:08:30 [Q] ERROR Failed [mars-burger-mountain-helium] - Model class django.contrib.sessions.models.Session doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS. : Traceback (most recent call last):
  File "/home/inventree/env/lib/python3.9/site-packages/django_q/cluster.py", line 436, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/home/inventree/src/InvenTree/InvenTree/tasks.py", line 240, in delete_expired_sessions
    from django.contrib.sessions.models import Session
  File "/home/inventree/env/lib/python3.9/site-packages/django/contrib/sessions/models.py", line 10, in <module>
    class Session(AbstractBaseSession):
  File "/home/inventree/env/lib/python3.9/site-packages/django/db/models/base.py", line 113, in __new__
    raise RuntimeError(
RuntimeError: Model class django.contrib.sessions.models.Session doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```
`/server.err.log`: no errors.

**Comment by matmair** [comment]
What do you exactly? The linked issue consistent of multiple errors. If you mean css is not loading try running 

inv Update
inv static

to refresh the static content and check if your proxy is serving any content at /static/

**Comment by matmair** [comment]
The background worker has nothing to do with the interface, we just need it for long running actions that would block the interface

**Comment by JustusRijke** [comment]
> What do you exactly? The linked issue consistent of multiple errors. 

Created a fresh VM, upgraded to PostgreSQL 14, followed all steps in https://inventree.readthedocs.io/en/latest/start/install/ and https://inventree.readthedocs.io/en/latest/start/production/ 

Interface is then shown exactly as the screenshot in https://github.com/inventree/InvenTree/issues/1720#issue-929392714

I starting the background worker manually as a somewhat random tryout as it seemed to be related https://github.com/inventree/InvenTree/issues/1720#issuecomment-868500904.

`inv static` did copy files (see below), but the interface remains the same.

```
(env) inventree@inventreeserver:~/src$ inv static
render label.js
render plugin.js
render model_renderers.js
render part.js
render attachment.js
render api.js
render company.js
render build.js
render filters.js
render modals.js
render bom.js
render barcode.js
render helpers.js
render order.js
render stock.js
render report.js
render forms.js
render table_filters.js
render tables.js
rendered all files in /home/inventree/src/InvenTree/templates/js/translated

399 static files copied to '/home/inventree/data/static', 2250 unmodified.
```

**Comment by JustusRijke** [comment]
> check if your proxy is serving any content at /static/

Can you explain how I can check this?

**Comment by matmair** [comment]
@JustusRijke just open a web browser and open the address you configured InvenTree for with `/static` appended. So if you are deploying on your local machine with default settings that would be `127.0.0.1:8000/static/`

**Comment by JustusRijke** [comment]
@matmair 127.0.0.1:8000/static redirects to the login page

**Comment by matmair** [comment]
> @matmair `127.0.0.1:8000/static` redirects to `127.0.0.1:8000/accounts/login/?next=/static`, after login it redirects to `127.0.0.1:8000/index/`
> 
Please check with `127.0.0.1:8000/static/` as written above. The last backslash is important as it identifies the static directory that seems to make the problems you experience. This subpath should be served via nginx, apache2 or any other proxy as described in the docs. It also is the most prominent source of error with production environments.

**Comment by JustusRijke** [comment]
@matmair thanks for all your quick replies!
I just deployed a new server (Ubuntu 21.04) without doing any upgrades (only the security updates). Now the page loads correctly.
So either I made a mistake somewhere during the previous install (whoops), or the Ubuntu upgrades / PostgreSQL 14 broke something. I'll try to reproduce this.

Two notes:
1. `psycopg2` package is missing in `requirements.txt`
2. https://inventree.readthedocs.io/en/latest/start/install/#configure-inventree-options asks the user to edit `config.yaml`. Before this step, `config_template.yaml` should be renamed to `config.yaml`
